variables.tf

```diff
@@ -204,3 +204,21 @@ variable "ca_cert_identifier" {
   description = "The identifier of the CA certificate for the DB instance"
   default     = null
 }
+
+variable "ssm_parameter_enabled" {
+  type        = bool
+  default     = false
+  description = "Whether an SSM parameter store value is created to store the key's private key pem."
```

Comment:
```suggestion
  description = "Whether an SSM parameter store value is created to store the database password."
```

Reply:
sorry @kevcube, i forgot to edit this one. i will fix it